### DISCUSSION TITLE
Please try again later. If you are using a VPN, try turning it off.

### DISCUSSION THREAD

**Discussion by namezzz** [x-dr/chatgptProxyAPI#54] [Q&A]
使用文档介绍的方法，请求openai接口的时候回返回以下页面。
![image](https://github.com/x-dr/chatgptProxyAPI/assets/36131103/d583f486-0771-432e-90a5-1c5ae1a50254)

**Comment by x-dr** [comment]
一切正常 ，而且api请求不会出现你那图片的情况

<img width="974" alt="image" src="https://github.com/x-dr/chatgptProxyAPI/assets/79109673/f1436c5a-a563-4e6d-9a90-4ffc6c81aa7a">

**Comment by duanchao1990** [comment]
请求错误: 401 Client Error: Unauthorized for url: https://xxx.dev/v1/chat/completions
